docs/vsc_calcua.md

```diff
@@ -0,0 +1,255 @@
+
+3. Choose whether you want to run in [local execution mode on a single node](#local-nextflow-run-on-a-single-interactive-node) or make use of the [Slurm job scheduler to queue individual pipeline tasks](#schedule-nextflow-pipeline-using-slurm).
+
+    - For Slurm scheduling, choose a partition profile ending in `*_slurm`. E.g., `nextflow run pipeline -profile vsc_calcua,broadwell_slurm`.
+    - For local execution mode on a single node, choose a partition profile ending in `*_local`. E.g., `nextflow run pipeline -profile vsc_calcua,broadwell_local`.
+
+    Note that the `-profile` option can take multiple values, the first one always being `vsc_calcua` and the second one a partition plus execution mode.
+
+4. Specify the _partition_ that you want to run the pipeline on using the [`sbatch` command's `--partition=<name>` option](https://docs.vscentrum.be/jobs/job_submission.html#specifying-a-partition) and how many _resources_ should be allocated. See the [overview of partitions and their resources](#overview-of-partition-profiles-and-resources) below, or refer to [the CalcUA documentation](https://docs.vscentrum.be/antwerp/tier2_hardware.html) for more info.
+
+    - For Slurm scheduling, the partition on which the head process runs has no effect on the resources allocated to the actual pipeline tasks. The head process only requires minimal resources (e.g., 1 CPU and 4 GB RAM).
+    - For local execution mode on a single node, the partition selected via `sbatch` must match the one selected with nextflow's `-profile` option, otherwise the pipeline will not launch. It is probably convenient to simply request a full node (e.g., `--cpus-per-task=28` and `--mem=112G` for broadwell). Omitting `--mem-per-cpu` or `--mem` will [allocate the default memory value](https://docs.vscentrum.be/jobs/job_submission.html#requesting-memory), which is the total available memory divided by the number of cores, e.g., `28 * 4 GB = 112 GB` for broadwell (`128 GB - 16 GB buffer`).
+
+5. Submit the job script containing your full `nextflow run` command via `sbatch` or from an an interactive `srun` session launched via `screen` or `tmux` (to avoid the process from stopping when you disconnect your SSH session).
+
+---
+
+## Location of output and work directory
+
+All of the intermediate files required to run the pipeline will be stored in the `work` directory. It is recommended to delete this directory after the pipeline has finished successfully because it can get quite large, and all of the main output files will be saved in the `results/` directory anyway.
+
+This config contains a `cleanup` command that removes the `work` directory automatically once the pipeline has completed successfully. If the run does not complete successfully then the `work` directory should be removed manually to save storage space. The default work directory is set to `$VSC_SCRATCH/work` per this configuration.
```

Comment:
> These docs are amazing!

Thank you!

> Could mention nextflow clean -f command for manual clean up?

Oh, excellent suggestion, done!

> but why not let nextflow select the partition for each job based on the task requirements , rather than a user trying to 'guess' the partition themselves?

Honestly, because I straight up didn't think of it! The standard modus operandi on our HPC is to log in to one of the main clusters (each containing different partitions) depending on job requirements and/or availability, so that's the approach I ended up doing using here too.

I do like your suggestion though, so let's see...

The main clusters differ in available nodes (28-64), available memory and wall time, and then there are a small number of specialized nodes (like high memory or GPU nodes). One thing I'd be a bit worried about is that choosing the appropriate queue based on task requirements could be a bit error prone or lead to situations where a particular partition is under-utilized and tasks are waiting a long time in the queue of another type. E.g., if I add something like the following:

```
queue = '{ task.cpus > 28 ? 'zen2,zen3' : 'broadwell' }
```

Would that not mean that only tasks which request more than 28 threads (which I assume are quite rare?) would end up on the zen nodes? In reality, it might be faster if multiple tasks are allocated to a single zen node, rather than all waiting in the broadwell queue. So I guess I could add those as a fallback, even if they might be "over-specced":

```
queue = '{ task.cpus > 28 ? 'zen2,zen3' : 'broadwell,zen2,zen3' }
```

Adding in memory and wall time differences, I end up with this massive conditional statement:

```
queue = {
                // long running
                if ( task.time > 3.day) {
                    'skylake'
                // high memory
                } else if ( task.memory > 240.GB ) {
                    'zen3_512'
                // medium memory and high cpu
                } else if ( task.memory > 112.GB && task.cpus > 28 ) {
                    'zen2,zen3'
                // medium memory and lower cpu
                } else if ( task.memory > 112.GB && task.cpus < 28 ) {
                    'broadwell_256,zen2,zen3'
                // lower memory and high cpu
                } else if ( task.cpus > 28 ) {
                    'zen2,zen3'
                // lower memory and lower cpu
                } else {
                    'broadwell,skylake,zen2,zen3'
                }
            }
```

I'll add this in as an additional generic "slurm" profile, and see how it turns out!